modules/ocf_stats/manifests/grafana.pp

```diff
@@ -0,0 +1,62 @@
+# grafana config
+class ocf_stats::grafana {
+  class { 'ocf::packages::grafana::apt':
+    stage => first,
+  }
+
+  $canonical_url = $::host_env ? {
+    'dev'  => 'dev-prometheus.ocf.berkeley.edu',
+    'prod' => 'prometheus.ocf.berkeley.edu',
+  }
+
+  class { 'grafana':
+    install_method           => 'repo',
+
+    # Unfortunately, puppet-grafana requires that we specify a version number
+    # here.
+    version                  => '5.2.1',
+    manage_package_repo      => false,
+
+    cfg                      => {
+      app_mode         => 'production',
+      server           => {
+        http_addr => '127.0.0.1',
+        http_port => 8990,
+        domain    => $canonical_url,
+        root_url  => "https://${canonical_url}/grafana/",
+      },
+      database         => {
+        type => 'sqlite3',
+        path => 'grafana.db',
```

Comment:
What kind of data does this store? While this could be using mysql, I think if you want do it this way instead of using docker, then using sqlite is probably fine. Does this data need to be backed up at all?